### ISSUE TITLE
Help & FAQs page

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#191]
For the Help & FAQs page, it should identify FAQS as abbreviation and read "Frequently asked questions".

**Comment by Rajat-Sen** [comment]
Also, it seems like we are using different font size as compared to rest of Settings screen on this page.

**Comment by ForumG-SC** [comment]
> Also, it seems like we are using different font size as compared to rest of Settings screen on this page.

Text can be increased from the settings for this page

**Comment by ForumG-SC** [comment]
Done

**Comment by ForumG-SC** [comment]
@Rajat-Sen As per our conversation, this should be changed to Help and Frequently asked questions.

**Comment by Rajat-Sen** [comment]
It's been partially fixed, the FAQ button at the bottom of the screen is still being read as FAQ instead of "Frequently asked questions".

**Comment by Rajat-Sen** [comment]
Validated!
